AWS/legos/aws_get_ec2_instance_age/aws_get_ec2_instance_age.py

```diff
@@ -0,0 +1,103 @@
+##  Copyright (c) 2022 unSkript, Inc
+##  All rights reserved.
+##
+## written by Doug Sillars with the aid of ChatGPT
+##read the blog https://unskript.com/will-ai-replace-us-using-chatgpt-to-create-python-actions-for-unskript/
+##
+
+from typing import List, Dict
+from pydantic import BaseModel, Field
+import pprint
+from datetime import datetime, timezone, timedelta
+from unskript.connectors.aws import aws_get_paginator
+
+
+from beartype import beartype
+@beartype
+def aws_get_ec2_data_traffic_v2_printer(output):
+    if output is None:
+        return
+    pprint.pprint(output)
+
+
+class InputSchema(BaseModel):
+    region: str = Field(
+        title='Region',
+        description='AWS Region of the ECS service')
+
+
+
+@beartype
+def aws_get_ec2_data_traffic_v2(handle, region: str) -> Dict:

```

Comment:
wrong function. Should be `aws_get_ec2_instance_age`

Reply:
fixed